cmd/crawler/main.go

```diff
@@ -107,8 +108,14 @@ func postgresPersister(config *utils.CrawlerConfig) *persistence.PostgresPersist
 		log.Errorf("Error connecting to Postgresql, stopping...; err: %v", err)
 		os.Exit(1)
 	}
-	// Attempts to create all the necessary tables here
-	err = persister.CreateTables()
+	// Create version_data table
+	err = persister.CreateVersionTable(&config.VersionNumber)
+	if err != nil {
+		log.Errorf("Error creating tables, stopping...; err: %v", err)
+		os.Exit(1)
+	}
+	// Create event table
+	err = persister.CreateEventTable()
```

Comment:
Let's go with this, but i was actually thinking that we could have the `CreateVersionTable()`, then all other tables would be created as we had before, under `CreateTables()`.